scripts/build.js

```diff
@@ -46,13 +47,19 @@ const cssDecodeUnicode = (value) => {
   return value.replace('&#x', '\\').replace(';', '').toLowerCase();
 };
 
+const icons = await getIconsData();
+const iconKeys = icons.map((icon) => {
+  const slug = icon.slug ?? titleToSlug(icon.title);
```

Comment:
You've used OR in *build-testpage.js*

```suggestion
  const slug = icon.slug || titleToSlug(icon.title);
```

Reply:
Thanks, let's me them consistent.